### DISCUSSION TITLE
option to make comments that aren't on code resolvable conversations

### DISCUSSION THREAD

**Discussion by DetachHead** [community/community#9540] [Pull Requests]
currently only comments on code are "resolvable". on gitlab, regular comments can be marked as "threads" which can be resolved as well

![image](https://user-images.githubusercontent.com/57028336/147877761-1f181a1f-3b2b-428e-aadf-228660768462.png)

**Comment by alexgarel** [comment]
This is really needed, or comment about a general bug (for example something that's missing) get lost.

**Comment by mochadwi** [comment]
looks like similar and related(?)

1. https://github.com/community/community/discussions/8469
2. https://github.com/community/community/discussions/5633

**Reply by DetachHead** [reply]
yeah. however it looks like those ones don't explicitly specify that the threads need to be resolvable

**Comment by MatthewFrench** [comment]
This feature would be so nice.
